### DISCUSSION TITLE
Data card Figma prop type

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#515] [Proposals]
La implementación actual de la `dataCard` en web permite cambiar el tipo de asset entre los siguientes valores: `icon`, `image` o `none`

![Screenshot 2022-06-27 at 09 34 42](https://user-images.githubusercontent.com/44420072/175885109-c1a3f633-4647-404c-b4c0-8cc1243c7fd7.png)

Recientemente se ha añadido este approach a la `snapCard` donde nos encontramos con la misma prop:

![Screenshot 2022-06-27 at 09 36 25](https://user-images.githubusercontent.com/44420072/175885449-0f50434f-0ef0-40c1-a983-ed5715fee31d.png)

Debería la `dataCard` en Figma permitir este cambio aunque actualmente no aparezca contemplado dentro de sus specs?

**Comment by yceballost** [comment]
Tiene sentido! E igual estaría bien que en las specs aparezca esta posibilidad, no?

Qué opina @AnaMontes11?

**Reply by AnaMontes11** [reply]
Sí, me parece que tiene todo el sentido y además ganamos en coherencia entre diseño/código y cards.
